tapdb/universe_stats.go

```diff
@@ -285,16 +285,16 @@ type UniverseStats struct {
 
 	clock clock.Clock
 
-	statsMtx         sync.Mutex
+	statsMtx         sync.RWMutex
```

Comment:
nit: probably don't need to change this one, since we're not using any read-only locking for it.